processing/data_collection/gazette/spiders/rn_natal.py

```diff
@@ -0,0 +1,36 @@
+from datetime import datetime
+
+import dateparser
+from scrapy import Request, FormRequest
```

Comment:
```suggestion
from scrapy import FormRequest
```
This can be removed since it's not being used.